tests/catch/include/hip_test_common.hh

```diff
@@ -329,11 +329,89 @@ struct MemTraits<MemcpyAsync> {
   }
 };
 
-}  // namespace HipTest
 
+namespace {
```

Comment:
The changes to this file will have to be removed since the same code will be commited in a separate PR